wsr-tic-tac-toe/src/__tests__/game-board.test.tsx

```diff
@@ -0,0 +1,226 @@
+        const buttonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await buttonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('0');
+        expect(await inputDriver.getErrorMessage()).toEqual(BOARD_SIZE_MORE_THAN_ZERO);
+    });
+
+    it('should show "Player1\'s turn" after creating a game with boardSize: 2', async () => {
+        const buttonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await buttonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+        const turnOrderHeadingDriver = HeadingTestkit({ wrapper, dataHook: TURN_ORDER_HEADING });
+        expect(await turnOrderHeadingDriver.exists()).toBeTruthy();
+        expect(await turnOrderHeadingDriver.getText()).toEqual("Player1's turn");
+    });
+
+    it('should show "Reset game" button after creating a game with boardSize: 2', async () => {
+        const createGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await createGameButtonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+
+        const resetGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: RESET_GAME_BUTTON,
+        });
+        expect(await resetGameButtonDriver.exists()).toBeTruthy();
+    });
+
+    it('should show 4 squares after creating a game with boardSize: 2', async () => {
+        const createGameButtonDriver = ButtonTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_BUTTON,
+        });
+
+        await createGameButtonDriver.click();
+        const inputDriver = InputWithLabelTestkit({ wrapper: wrapper, dataHook: BOARD_SIZE_INPUT });
+
+        await inputDriver.enterText('2');
+        const messageBoxDriver = MessageBoxFunctionalLayoutTestkit({
+            wrapper: wrapper,
+            dataHook: CREATE_GAME_MESSAGE_BOX,
+        });
+        await messageBoxDriver.clickOnConfirmationButton();
+
+        const square0 = document.getElementById('0');
+        const square1 = document.getElementById('1');
```

Comment:
you should not mix testing utilities. If you are using testing-library, don't use plain document (since the testing library provides all the same tools)